spaceship.zsh

```diff
@@ -525,6 +533,30 @@ spaceship_hg() {
     "$SPACESHIP_HG_SUFFIX"
 }
 
+# EMBER
+# Show current version of ember, exception system.
+spaceship_ember() {
```

Comment:
It's as useful as seeing your Node.js version when working with Node, seeing the version there lets you know:

1. Ember is present since the version is present
2. The version number without having to run a command to see it

This is most useful when switching between projects with different versions, like when writing Ember addons, especially so since Ember releases new versions constantly.